CMakeLists.txt

```diff
@@ -4,7 +4,9 @@ cmake_minimum_required(VERSION 2.8.5)
 PROJECT( RTABMap )
 SET(PROJECT_PREFIX rtabmap)
 
-# Catkin doesn't support multiarch library path, 
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Remove this, it is already added with Torch is found here: https://github.com/introlab/rtabmap/blob/da8e76ffed836fad3319716e5e15112a59e4e4f5/CMakeLists.txt#L607